Misc/NEWS.d/next/Library/2022-12-28-00-28-43.gh-issue-100562.Hic0Z0.rst

```diff
@@ -0,0 +1 @@
+Improve performance of :meth:`pathlib.Path.absolute`.
```

Comment:
- State just how significant the performance improvement is
- Lay out more explicitly the trade-off here

```suggestion
Improve performance of :meth:`pathlib.Path.absolute` by nearly 2x. This comes
at the cost of a performance regression in :meth:`pathlib.Path.cwd`, which is
generally used less frequently.
```